VMWare/vmware-esxi/tests/AUTH_SSH_event_closed.json

```diff
@@ -16,6 +16,9 @@
       "product": "ESXi",
       "vendor": "VMware"
     },
+    "process": {
+      "name": "vob.user.ssh.session.closed"
+    },
```

Comment:
Can you fix the `EVENT_LOG_SSH_AUTH` grok? The `process` capture is misnamed. Its value must land in event.action.